requirements.txt

```diff
@@ -1,30 +1,30 @@
 # Optionals
-itsdangerous
-jinja2
-python-multipart
-pyyaml
-requests
+itsdangerous==2.0.1
+jinja2==3.0.3
+python-multipart==0.0.5
+pyyaml==6.0
+requests==2.26.0
```

Comment:
I guess we should do the same as this: https://github.com/encode/httpx/pull/1721#discussion_r660772826

Maybe we can use `-e .[full]`?

Reply:
Oh right, we already have those.